tensorflow/contrib/data/python/kernel_tests/writer_ops_test.py

```diff
@@ -52,6 +53,7 @@ def _createFile(self, options=None):
     for i in range(self._num_records):
       writer.write(self._record(i))
     writer.close()
+    sys.stdout.flush()
```

Comment:
What's the intention of flushing `stdout` here? As far as I can tell, nothing in this function writes to `stdout`, so I wouldn't expect any change in behavior.

Reply:
so during reading we would check the size of the file first. if the file is not closed, it would give size 0. which the reader would `raise OutOfRange` . 

since `self._createFile` has already created the file, i'm not sure why we need to use the session to run writer again? 
https://github.com/tensorflow/tensorflow/blob/d64a50db3e5616da63049ce778cb3d1ece4c0ed8/tensorflow/contrib/data/python/kernel_tests/writer_ops_test.py#L80 sys.stdout.flush() is a mistake here. just want to flush or close the file writer has written, so that reading won't raise Exceptions.